vxtelegram/telegram.py

```diff
@@ -187,12 +177,19 @@ def handle_raw_inbound_message(self, message_id, request):
             message_id=message_id,
             content=message['content'],
             to_addr=message['to_addr'],
+            to_addr_type='telegram_username',
             from_addr=message['from_addr'],
+            from_addr_type='telegram_id',
```

Comment:
@hodgestar a `telegram_username` is the human readable "@user" that's displayed in chats, whereas a `telegram_id` is an integer that identifies the user (which is what the Telegram API uses). Wherever a user's identity isn't going to be used by the API, I've rather used the `telegram_username`, simply because it makes more sense in the logs (like our bot username, which is never used in any requests).